app/src/features/projects/view/components/ProjectObjectives.tsx

```diff
@@ -15,19 +21,58 @@ export interface IProjectObjectivesProps {
  */
 const ProjectObjectives: React.FC<IProjectObjectivesProps> = (props) => {
   const {
-    projectForViewData: { objectives }
+    projectForViewData: { objectives, projectId }
   } = props;
 
+  const history = useHistory();
+
+  const openModalEdit = () => {
```

Comment:
tests for all these probably need to be added as well, but not a blocker for this PR